<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendSmsDialog</class>
 <widget class="QDialog" name="SendSmsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Send SMS</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Sendto:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="AllContactsRadioButton">
       <property name="text">
        <string>All their contacts</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="NumberListRadioButton">
       <property name="text">
        <string>Number in list</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Content:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditContent"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textEditContent</tabstop>
  <tabstop>AllContactsRadioButton</tabstop>
  <tabstop>NumberListRadioButton</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>listWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>AllContactsRadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>listWidget</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NumberListRadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>listWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
